app/routers/users.py

```diff
@@ -215,10 +228,27 @@ async def delete_user(id: str):  # pragma: no cover
 @router.get(
     "/{id}/accounts",
     dependencies=[Depends(authentication_required)],
-    response_model=list[AccountUserRead],
+    response_model=LimitOffsetPage[AccountUserRead],
```

Comment:
I think this endpoint should return an Account not AccountUser.

Reply:
Forget to mention that of course we need information about the account user but it should be nested